src/services/messengerSync/utils.js

```diff
@@ -149,9 +147,16 @@ const buildAddUserMessage = async (data) => {
 };
 
 const messengerActivatedForSchool = async (data) => {
```

Comment:
```suggestion
const messengerIsActivatedForSchool = async (data) => {
```

are there no permissions to check? i would suggest replace data with` {userid=null, schoolId}`